xapian-applications/omega/omega.cc

```diff
@@ -593,7 +626,9 @@ try {
     // topdoc+max(hits_per_page+1,min_hits)
     val = cgi_params.find("MINHITS");
     if (val != cgi_params.end()) {
-	min_hits = atol(val->second.c_str());
+	if (!parse_unsigned(val->second.c_str(),min_hits)) {
+	    throw "MINHITS parameter must be > 0\n";
```

Comment:
I think this should be `>=` - we don't reject zero (either currently or in your patch), the documentation doesn't suggest zero is invalid, and zero is actually the default value if you don't specify one so allowing it through is just like not specifying it, which seems logical.

Reply:
oh sorry, missed that. will fix it.